.github/workflows/smoketest-chart.yaml

```diff
@@ -0,0 +1,116 @@
+name: Helm Chart Smoketest
+
+on: [push, workflow_dispatch, pull_request]
+
+env:
+  K8S_VERSION: v1.32.3
+  MICROK8S_CHANNEL: 1.32/stable
+
+jobs:
+  helm-install-smoke-test:
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        config:
+          - {
+              type: "kind"
+            }
+          - {
+              type: "minikube"
+            }
+          - {
+              type: "microk8s"
+            }
+          - {
+              type: "k3d"
+            }
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Install helm
+        uses: Azure/setup-helm@v4
+        with:
+          version: v3.18.0
+
+      # Note: 'uses' doesn't support variable interpolation, hence the
+      # k8s-specific steps below.
+      # Ref: https://github.com/orgs/community/discussions/25824
+      - name: Create kind cluster
+        if: matrix.config.type == 'kind'
+        uses: helm/kind-action@v1
+        with:
+          cluster_name: kind
+          version: v0.27.0
+          node_image: kindest/node:${{ env.K8S_VERSION }}
+
+      - name: Create minikube cluster
+        if: matrix.config.type == 'minikube'
+        uses: medyagh/setup-minikube@v0.0.19
+        with:
+          container-runtime: containerd
+          kubernetes-version: ${{ env.K8S_VERSION }}
+
+      - name: Create microk8s cluster
+        if: matrix.config.type == 'microk8s'
+        uses: balchua/microk8s-actions@v0.4.3
+        with:
+          channel: ${{ env.MICROK8S_CHANNEL }}
+          addons: '["hostpath-storage"]'
+
+      - name: Create k3d cluster
+        if: matrix.config.type == 'k3d'
+        uses: AbsaOSS/k3d-action@v2
```

Comment:
These are a lot of third-party actions fwiw. Given this is a test-only change, I think it is fine, but don't know that we have that granular of permissioning/access controls to prevent token leakage. I wonder if we should have a second category of test-only repository that lacks any tokens and just gets code pushed into it, to trigger pipelines which use third-party actions? My 2c.

Reply:
I get your concerns, but also dont have an immediate solution to this. A separate repo possibly mitigates the raised concerns, but also seems like a pretty unconventional way.

From my PoV we could also keep this PR open and discuss general approach (how to deal with external actions) in the WG Supply Chain Security group.

In the meantime I will try to
* "audit" the used libraries
* pin them on their version
* check whether we could work with a more permissive Github Token.

Reply:
> but also seems like a pretty unconventional way.

Agreed, I think it is perhaps the GitLab way leaking out ;-) 

The first one seems actively maintained, but the latter two seem to have stagnated recently. That could just mean they're finished, but not sure.

Reply:
Just my from my pov testing the chart only with `kind` should be sufficient. It also seems to be the one with a well-maintained action.

It's very likely that our users want to deploy the chart on production-ready flavors like AKS, EKS, OpenShift, Rancher, ... 
So I'm not sure testing on all of those edge/local Kubernetes flavor provide much added value. Testing agains one of them is probably sufficient to verify that the Helm chart works.

Reply:
Agreed. I reduced the test to kind only and rebased the PR.

Happy for another review <3